src/ngraph/builder/quantization_util.hpp

```diff
@@ -95,11 +96,33 @@ namespace ngraph
                 // s32 = f32 * std::pow(2, 31)/ max_abs32;
                 // s8 = f32 * std::pow(2, 7)/ max_abs8;
                 // s8 = s32 * std::pow(2, -24) * max_abs32 / max_abs8;
-                const float scale = static_cast<float>(
-                    (std::pow(2, -24) * static_cast<double>(max_abs32 / max_abs8)));
+                float scale = static_cast<float>((std::pow(2, is_output_signed ? -24 : -23) *
```

Comment:
comments not updated to reflect signed case.  could you add some comments as to why this math makes sense?  I don't have any idea why this calculation does what it does other than "that's what TensorFlow does".